src/clang.rs

```diff
@@ -37,7 +37,7 @@ impl Cursor {
         };
 
         // Try to undo backend mangling
-        if cfg!(target_os = "macos") || cfg!(target_os = "windows") {
+        if cfg!(target_os = "macos") {
```

Comment:
IIRC this hack was there for windows-gnu, instead of MSVC, can we use an specific msvc cfg flag to skip this, or otherwise leave a FIXME comment?